script/Common/00_00.txt

```diff
@@ -907,8 +911,9 @@ I heard a woman's voice, clear as a bell.
 [sha:7e9b8e7482cd8cd24f727750df48740712055da7]{
 -- Page 60, Offset 281.
 -- ただでさえちっこいんだから、草むらでうずくまってると見えないのよね。気をつけなさい、あやうく蹴り飛ばされるところだったんだから」
+// 'into you' then why does shiki ask "who" later? have to be ambigious for this exchange to even make sense.
```

Comment:
I'd keep this `into you`. Doesn't really follow to me that `through here` -> `through who`, and I think him not understanding `you` works with his whole "I am going insane, what is the self" mindset

Reply:
I do still stand by the fact that it makes 0 sense in English at the moment. It needs to change.

The exchange is very Japanese, since she says 蹴り飛ばされる in Japanese in the passive mood, so it almost is appropriate to ask "by who"?

That said, perhaps this needs more thorough editing:

> I almost plowed right into you.

> You almost plowed right into... me?

> Yeah, isn't that obvious?

> You and I are the only ones here, who else besides you could it be?"

Obviously quite a departure, but something has to be done with this exchange

Reply:
Another possibility (closer to original): 

>You almost got yourself plowed over.

>Plowed over by... who?

>What, isn't it obvious?

>You and I are the only ones here, so besides me could it possibly be?"

Reply:
Yeah reading your second one only now realizing that this exchange is actually pretty wrong. 
> けりとばされるって、誰**に**

you're right, he's asking _who nearly knocked into him_ rather than _who was it that was about to get knocked into_. I might prefer sent flying to plowed over here though, plow `into` is common but `over` not so much
> You need to be more careful, you nearly got sent flying.
> Sent flying...? By who?